pinot-core/src/main/java/com/linkedin/pinot/core/io/reader/impl/v1/BaseChunkSingleValueReader.java

```diff
@@ -51,13 +54,14 @@
    * Constructor for the class.
    *
    * @param pinotDataBuffer Data buffer
-   * @param decompressor Data decompressor
    */
-  public BaseChunkSingleValueReader(PinotDataBuffer pinotDataBuffer, ChunkDecompressor decompressor) {
-    _chunkDecompressor = decompressor;
+  public BaseChunkSingleValueReader(PinotDataBuffer pinotDataBuffer) {
```

Comment:
will be super useful if you can add a picture here, with different fiends in version 1 and beyond.

Reply:
Updated header layout in FixedByteChunkSingleValueWriter to explicitly mention which fields are in version 2 onwards.